src/fileCache.ts

```diff
@@ -0,0 +1,59 @@
+import * as vscode from "vscode";
+import * as path from "path";
+import * as fs from "fs";
+
+let fileCache: Map<string, string[]> = new Map();
+
+export function initializeFileCache() {
+  const workspaceFolders = vscode.workspace.workspaceFolders;
+  if (!workspaceFolders) {
+    return;
+  }
+
+  workspaceFolders.forEach(folder => {
+    recursiveSearch(folder.uri.fsPath, folder.uri.fsPath);
+  });
+}
+
+function recursiveSearch(dir: string, workspaceRoot: string) {
```

Comment:
We don't need this function. The vscode api provides a [findFiles](https://vscode-api.js.org/modules/vscode.workspace.html#findFiles) that can handle this

Reply:
perfect!